pyccl/concentration.py

```diff
@@ -0,0 +1,317 @@
+from . import ccllib as lib
```

Comment:
A general comment here. Do we want all of this code at the Python level? I understand we are python-first, but my guess is that eventually we want to use these routines to do halo model computations. It is very likely that those computations will happen mostly in C for performance reasons. In this case, I think we would want the code callable from C. This is possible with Python OFC but it would mean we are stuck with the GIL and so would not have an opportunity to use OpenMP.

Reply:
So here's my thinking: for halo-model computations we need to solve a few integrals over mass. All of these routines allow you to compute some of the functions of mass that go into those integrals. The idea then is that, when we write the general halo model module, these routines will generate these functions sampled over a range of log(Mass), we'll pass those to C, and C will do the integrals with GSL (and using openMP where needed).
Kind of the same rationale behind the generalized P(k)s right now. does this make sanse?

Reply:
Ahhhh great. This sounds good to me!

Reply:
:partyparrot: